src/charm.py

```diff
@@ -238,6 +245,16 @@ def _on_storage_detaching(self, event: StorageDetachingEvent) -> None:
 
     def _on_install(self, _) -> None:
         """Handler for `install` event."""
+        sysctl_yaml = yaml.safe_load(Path("./src/templates/sysctl.yaml").read_text())
+        sysctl = SysctlConfig(config_params=sysctl_yaml["testing"], app_name=CHARM_KEY)
```

Comment:
I think we should use current charm name, not sure if we can have two kafka on same machine, but for sure I can deploy kafka with different name and it would be more easier to know from which charm it's coming.